src/main/java/no/nav/foreldrepenger/los/reservasjon/ReservasjonRepository.java

```diff
@@ -47,6 +49,58 @@ order by coalesce(r.endretTidspunkt, r.opprettetTidspunkt) desc
             .getResultList();
     }
 
+    public List<SisteReserverteMetadata> hentSisteReserverteMetadata(String uid) {
+        /*
```

Comment:
Bør det være en test - mer for å sikre at spørringen blir lagt merke ifm refaktorering enn av funksjon.

Reply:
Har en liten test som gjør at query kjøres, men den er veldig basal og dekker ikke hele query så godt.

Reply:
jeg legger til en egen som eksplisitt refererer til query. Da dekker vi kjøringen og unngår at testen fjernes.